flink-kubernetes/src/main/java/org/apache/flink/kubernetes/KubernetesClusterDescriptor.java

```diff
@@ -91,7 +104,11 @@ public String getClusterDescription() {
 			}
 
 			try {
-				return new RestClusterClient<>(configuration, clusterId);
+				return new RestClusterClient<>(
+					configuration,
+					clusterId,
+					new StandaloneClientHAServices(HighAvailabilityServicesUtils.getWebMonitorAddress(
```

Comment:
It would be better to leave a comment here for why we use StandaloneClientHAServices for k8s client.

Reply:
I will add some comments here.